### PR TITLE
[s] fixes beanbag slug duplication

### PR DESCRIPTION
**Pull request by ToasterBiome**
# Document the changes in your pull request

alternatively i could just make a copy and then check it against itself and then we can put boxes back, but I think that would be too prone to error anyway. Request changes if you want it so.

# Changelog

:cl:  
tweak: boxes are no longer allowed to be re-added to wardrobes
/:cl:

### CODE REVIEW

**Comment by Bibby0110** [comment]
alternatively you port the proper fix from TG

**Comment by ToasterBiome** [comment]
> alternatively you port the proper fix from TG

Mind sending me it?

**Comment by Bibby0110** [comment]
> > alternatively you port the proper fix from TG
> 
> Mind sending me it?

https://github.com/tgstation/tgstation

**Comment by JamieD1** [comment]
> > > alternatively you port the proper fix from TG
> > 
> > 
> > Mind sending me it?
> 
> https://github.com/tgstation/tgstation

Amazing.

**Comment by ToasterBiome** [comment]
> > > alternatively you port the proper fix from TG
> > 
> > 
> > Mind sending me it?
> 
> https://github.com/tgstation/tgstation

The PR

**Comment by Bibby0110** [comment]
> > > > alternatively you port the proper fix from TG
> > > 
> > > 
> > > Mind sending me it?
> > 
> > 
> > https://github.com/tgstation/tgstation
> 
> The PR

yeah it's at that link

File: code/modules/vending/wardrobes.dm [hunk 1]
```diff
@@ -15,6 +15,8 @@
 			var/obj/item/radio/headset/HS = I
 			if(HS.keyslot != initial(HS.keyslot)) // Hey, you stole something!
 				return FALSE
+		if(istype(I,/obj/item/storage/box)) //no putting boxes back, sorry
```
Comment (JamieD1) [inline code/modules/vending/wardrobes.dm]:
```suggestion
		if(istype(I, /obj/item/storage/box)) //no putting boxes back, sorry
```

**Review by JamieD1 — APPROVED** [review]

**Pull request merged**